setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Install dependencies
+npm i -g quicktype
+
+# run transpiler
+quicktype --src schema.json \
+--src-lang json \
+--lang rust \
+--out workspace/gh-workflow-rs/src/model.rs
```

Comment:
should run `npm run generate` which should generate the `model.rs`. The generate command should call JS code.